src/index.js

```diff
@@ -1,22 +1,31 @@
-// react libraries
+// import react
```

Comment:
#### Nitpick
React and react-dom are `// react libraries` while you can comment react-redux as `// third-party libraries`

Reply:
Okay, I am on it. Thank you.